client/test/unit/lib/paymentCalc/84532.test.ts

```diff
@@ -71,7 +88,8 @@ describe("PaymentCalc: Base", () => {
     await axiom.prove(inputs);
     const args = axiom.getSendQueryArgs();
 
-    const queryCost = calculateQueryCost(CHAIN_ID, basefee, baseFeeScalar, blobBaseFee, blobBaseFeeScalar, maxFeePerGas, callbackGasLimit, proofVerificationGas);
+    const queryCost =  await calculateQueryCost(publicClient, CHAIN_ID, basefee, baseFeeScalar, blobBaseFee, blobBaseFeeScalar, maxFeePerGas, callbackGasLimit, proofVerificationGas);
+    expect(queryCost).toEqual(BigInt(args?.value ?? 0));
```

Comment:
Ok I've isolated the issue. This equality check was left as residue while I was testing for fixed block equality. The equality without fixing blocks is quite unstable.